.pre-commit-config.yaml

```diff
@@ -84,6 +84,4 @@ repos:
     name: Add License
     entry: python tools/add_copyright.py
     language: python
-    stages: [pre-commit]
```

Comment:
What does removing this line do? Don't we want it to happen in the pre-commit still?

Reply:
I think this is the default value. Changing it did not change behavior when I tried committing again. None of the other hooks specify this stage, so I removed this for parity and brevity.